app/services/retrieve_pipeline_viz_graph_data_service.rb

```diff
@@ -10,11 +10,12 @@ class RetrievePipelineVizGraphDataService
   #     - jobStatus: The job status of the stage
   #     - steps: Array of objects, each one defining a node in that stage's graph. This object
   #       is composed of:
-  #           - stepName: The name to be displayed for this step
-  #               - inputEdges: An array of indices that map to edges in the @edges array (see below), each
-  #                 edge being an input edge to the node.
-  #               - ouputEdges: An array of indices that map to edges in the @edges array, each edge being an
-  #                 output edge from the node
+  #           - name: The name to be displayed for this step. This is the out target name.
```

Comment:
To make it more clear: `This is the out target name` -> `Currently, this corresponds to the name of the output target (works as there is a single output target per step).`